css/todo.scss

```diff
@@ -0,0 +1,77 @@
+
+/* To-do List */
+#to-do-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center; }
+  
+  #add-task {
+    min-width: 300px;
+    min-height: 40px;
+    padding: 1rem;
+    font-size: 1.5rem; 
+    color: var(--secondary-300);
+    border:0;
+    outline:none;
+    border: 2px solid var(--primary-500);
+  }
+  
+  #add-task::placeholder {
+    /* padding: 1rem; */ }
+  
+  
+  .tasks{
+    list-style: none;
+    padding-left: 0;
+    /* padding: 1rem; */
+  
+    }
+  
+    .todo__heading{
+      color: var(--secondary-700);
+      font-size: 2.4rem;
+      font-weight: 700;
+    }
+  
+  
```

Comment:
Why the formatting isn't working? Can you look into how to auto-format the code when you save a file?

Reply:
Done in the latest one.